templates/zimrc

```diff
@@ -57,6 +57,9 @@ ztermtitle='%n@%m:%~'
 # Input
 #
 
+# Set to vi or emacs
+zinput_mode='emacs'
+
```

Comment:
I know we've been using configuration variables like this so far, but I prefer to start using zstyle ones from now on (and migrate the existing ones to zstyle format when we release version 1.0).

So here it would be something like `zstyle ':zim:input' mode 'emacs'`

@Eriner, thoughts?

Reply:
I have no strong feelings one way or another, provided they don't become overly-verbose abominations like I've seen elsewhere. The entire point to the simple variables is just that; simplicity. That being said, I think this helps to clean up some pollution and makes the varibles more structured and accessible.